gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/Oplog.java

```diff
@@ -158,6 +158,8 @@
    */
   private final PersistentOplogSet oplogSet;
 
+  private final boolean isStandByOplog;
```

Comment:
Where is this being set to true? Ok I see a constructor. Need to see when it is being called.

Reply:
we discussed it in review...